document-generator-prosecution/pom.xml

```diff
@@ -54,7 +54,7 @@
         <dependency>
             <groupId>uk.gov.companieshouse</groupId>
             <artifactId>document-generator-interfaces</artifactId>
-            <version>unversioned</version>
+            <version>5f53c6f</version>
```

Comment:
Shouldn't this stay as unversioned (to remain consistent with accounts)?

Reply:
Yeah, you're right, forgot to remove this before I committed the POM file. I'll change and commit.